climakitae/core/data_interface.py

```diff
@@ -559,12 +569,23 @@ class DataParameters(param.Parameterized):
         shorthand alias to DataInterface.geographies
     _geography_choose: dict
         shorthand alias to Boundaries.boundary_dict()
+    _warming_level_times: pd.DataFrame
+        shorthand alias to DataInterface.warming_level_times
     colormap: str
         default colormap to render the currently selected data
     scenario_options: list of strs
         list of available scenarios (historical and ssp) for selection
     variable_options_df: pd.DataFrame
         filtered variable descriptions for the downscaling_method and timescale
+    warming_level: array
```

Comment:
Could these be called `warming_levels` instead?

Reply:
leaving as-is for now but we can change in a future PR. I'm not sure which I prefer; we can discuss later if you feel particularly strong on this subject.